### PR TITLE
﻿Improve card payout specs

### PR DESCRIPTION
**Pull request by philip-wood-cko**
# Description of changes in PR

[//]: # 'Please add your description here'
Update specs for NAS card payouts, as agreed with Joe Graves.
- Fixes various inaccuracies in the spec
- General improvements to clarity and readability
- Fixes build-time errors

## Checklist

- [x] Added a changelog entry to the `changelog.md` file in either `abc_spec` or `nas_spec`
- [ ] Contacted the Tech Docs team about any corresponding guides that need to be updated for www.checkout.com/docs

**Contributing options**: Look at [our documentation](https://checkout.atlassian.net/wiki/spaces/PD/pages/2169506663/API+ref+publication+process) for options to contribute to the API reference.

### CODE REVIEW

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest.yaml [hunk 1]
```diff
@@ -15,49 +15,49 @@ properties:
...
         maxLength: 29
         minLength: 29
       amount:
         type: integer
-        description: |     
-          Amount to be paid out in minor units from the currency account. The exact format [depends on the currency](https://checkout.com/docs/four/resources/calculating-the-amount). 
-          
-          One of `source.amount` or `amount` must be provided
+        description: | 
+          The amount you want to pay out from your currency account. The exact format [depends on the currency](https://checkout.com/docs/four/resources/calculating-the-amount), which is determined by Checkout.com from the currency account defined in `source.id`
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest.yaml:30]:
```suggestion
          The amount you want to pay out from your currency account. The exact format [depends on the currency](https://checkout.com/docs/four/resources/calculating-the-amount), which is determined by Checkout.com from the currency account defined in `source.id`.
```

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest.yaml [hunk 2]
```diff
@@ -15,49 +15,49 @@ properties:
...
+
+          ***Conditional*** - you must specify either `source.amount` or `amount`
+        minimum: 0
   destination:
     $ref: '#/components/schemas/CardPayoutRequestDestination'
   amount:
     type: integer
     description: |
-      The payout destination amount in minor units. The exact format [depends on the currency](https://checkout.com/docs/four/resources/calculating-the-amount). 
+      The payout destination amount in minor units. The exact format [depends on the currency](https://checkout.com/docs/four/resources/calculating-the-amount) defined in `currency`
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest.yaml:39]:
```suggestion
      The payout destination amount in minor units. The exact format [depends on the currency](https://checkout.com/docs/four/resources/calculating-the-amount) defined in `currency`.
```

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest.yaml [hunk 3]
```diff
@@ -15,49 +15,49 @@ properties:
...
+      funds_transfer_type:
+        type: string
+        description: The funds transfer type assigned to your specific use case by Checkout.com and the card networks      
       purpose:
         type: string
-        description: Mandatory when `destination.issuing_country` is Argentina, Bangladesh, Egypt, or India.
+        description: |
+          The purpose of the payout
+        
+          ***Conditional*** - required when required when the [card metadata](https://api-reference.checkout.com/#operation/requestCardMetadata!c=200&path=issuer_country&t=response) `issuer_country` = `AR` (Argentina), `BD` (Bangladesh), `EG` (Egypt) or `IN` (India)
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest.yaml:60]:
```suggestion
          ***Conditional*** - required when the [card metadata](https://api-reference.checkout.com/#operation/requestCardMetadata!c=200&path=issuer_country&t=response) `issuer_country` = `AR` (Argentina), `BD` (Bangladesh), `EG` (Egypt) or `IN` (India)
```

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/01_CardPayoutRequest_DestinationTypeCardAccountHolderIndividual.yaml [hunk 4]
```diff
@@ -88,19 +113,29 @@ properties:
           - tax_id
       number:
         type: string
-        description: The payout destination owner's ID number
+        description: The payout destination owner's identification number
         maxLength: 25
       issuing_country:
         type: string
-        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the payout destination owner's ID country of issue
+        description: If applicable, [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the issuing country of the payout destination owner's identification
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/01_CardPayoutRequest_DestinationTypeCardAccountHolderIndividual.yaml:120]:
```suggestion
        description: If applicable, the [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the issuing country of the payout destination owner's identification
```

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/01_CardPayoutRequest_DestinationTypeCardAccountHolderIndividual.yaml [hunk 5]
```diff
@@ -88,19 +113,29 @@ properties:
...
         maxLength: 2
         minLength: 2
       date_of_expiry:
         type: string
-        description: The payout destination owner's ID expiry date. Format 'yyyy-MM-dd'
+        description: If applicable, the expiration date of the payout destination owner's identification. Format `yyyy-MM-dd`
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/01_CardPayoutRequest_DestinationTypeCardAccountHolderIndividual.yaml:125]: To be consistent, should it be either `YYYY-MM-DD` or `yyyy-mm-dd`?

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/02_CardPayoutRequest_DestinationTypeCardAccountHolderCorporate.yaml [hunk 6]
```diff
@@ -69,16 +98,16 @@ properties:
...
       issuing_country:
         type: string
-        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the payout destination owner's ID country of issue
+        description: If applicable, [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the issuing country of the payout destination owner's identification
         maxLength: 2
         minLength: 2
       date_of_expiry:
         type: string
-        description: The payout destination owner's ID expiry date. Format 'yyyy-MM-dd'
+        description: If applicable, the expiration date of the payout destination owner's identification. Format `yyyy-MM-dd`
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/02_CardPayoutRequest_DestinationTypeCardAccountHolderCorporate.yaml:110]: To be consistent, should it be either `YYYY-MM-DD` or `yyyy-mm-dd`?

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationTypes/01_CardPayoutRequest_DestinationTypeToken.yaml [hunk 7]
```diff
@@ -13,9 +13,10 @@ properties:
     pattern: ^(tok)_(\w{26})$
     maxLength: 30
     minLength: 30
-    description: The Checkout.com token (eg a card, wallet or token)
+    description: The Checkout.com token (e.g., a card token)
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationTypes/01_CardPayoutRequest_DestinationTypeToken.yaml:16]:
```suggestion
    description: The Checkout.com token (for example, a card token)
```
Reason for suggestion: screen readers can misread 'e.g.' as 'egg'. 🥚

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationTypes/02_CardPayoutRequest_DestinationTypeId.yaml [hunk 8]
```diff
@@ -13,9 +13,10 @@ properties:
     pattern: ^(src)_(\w{26})$
     maxLength: 30
     minLength: 30
-    description: The payment source identifier (eg a card source identifier)
+    description: The payment source identifier (e.g., a card source identifier)
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationTypes/02_CardPayoutRequest_DestinationTypeId.yaml:16]:
```suggestion
    description: The payment source identifier (for example, a card source identifier)
```

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/01_CardPayoutRequest_SenderAccountHolderIndividual.yaml [hunk 9]
```diff
@@ -13,57 +14,72 @@ properties:
...
-        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the sender address
+        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the sender's residential address
         maxLength: 2
         minLength: 2
   reference:
     type: string
-    description: The reference of the sender. Only alphanumeric characters are allowed. Mandatory for Money Transfer and Third-Party use cases
-    maxLength: 16
+    description: |
+      The unique identifying reference of the sender (e.g., customer number)
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/01_CardPayoutRequest_SenderAccountHolderIndividual.yaml:76]:
```suggestion
      The unique identifying reference of the sender (for example, customer number)
```

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/02_CardPayoutRequest_SenderAccountHolderCorporate.yaml [hunk 10]
```diff
@@ -12,50 +13,61 @@ properties:
...
-        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the sender address
+        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the sender's registered address
         maxLength: 2
         minLength: 2
   reference:
     type: string
-    description: The reference of the sender. Only alphanumeric characters are allowed. Mandatory for Money Transfer and Third-Party use cases
-    maxLength: 16
+    description: |
+      The unique identifying reference of the sender (e.g., customer number)
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/02_CardPayoutRequest_SenderAccountHolderCorporate.yaml:64]:
```suggestion
      The unique identifying reference of the sender (for example, customer number)
```

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/02_CardPayoutRequest_SenderAccountHolderCorporate.yaml [hunk 11]
```diff
@@ -89,14 +109,14 @@ properties:
           - tax_id
       number:
         type: string
-        description: The sender's ID number. Mandatory when sending cross-border transactions from AU
+        description: The sender's identification number
       issuing_country:
         type: string
-        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the sender's country of issue. Mandatory when sending cross-border transactions from AU
+        description: If applicable, [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the issuing country of the sender's identification
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/02_CardPayoutRequest_SenderAccountHolderCorporate.yaml:115]:
```suggestion
        description: If applicable, the [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the issuing country of the sender's identification
```

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/02_CardPayoutRequest_SenderAccountHolderCorporate.yaml [hunk 12]
```diff
@@ -89,14 +109,14 @@ properties:
...
         minLength: 2
         maxLength: 2
       date_of_expiry:
         type: string
-        description: The sender's ID expiry date. Format 'yyyy-MM-dd'
+        description: If applicable, the expiration date of the sender's identification. Format `yyyy-MM-dd`
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/02_CardPayoutRequest_SenderAccountHolderCorporate.yaml:120]:
```suggestion
        description: If applicable, the expiration date of the sender's identification. Format `yyyy-MM-dd`.
```

To be consistent, should it be either `YYYY-MM-DD` or `yyyy-mm-dd`?

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/03_CardPayoutRequest_SenderAccountHolderGovernment.yaml [hunk 13]
```diff
@@ -1,57 +1,73 @@
...
-        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the sender address
+        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the sender's registered address
         maxLength: 2
         minLength: 2
   reference:
     type: string
-    description: The reference of the sender. Only alphanumeric characters are allowed. Mandatory for Money Transfer and Third-Party use cases
-    maxLength: 16
+    description: |
+      The unique identifying reference of the sender (e.g., customer number)
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/03_CardPayoutRequest_SenderAccountHolderGovernment.yaml:64]:
```suggestion
      The unique identifying reference of the sender (for example, customer number)
```

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/03_CardPayoutRequest_SenderAccountHolderGovernment.yaml [hunk 14]
```diff
@@ -85,14 +109,14 @@ properties:
           - tax_id
       number:
         type: string
-        description: The sender's ID number. Mandatory when sending cross-border transactions from AU
+        description: The sender's identification number
       issuing_country:
         type: string
-        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the sender's country of issue. Mandatory when sending cross-border transactions from AU
+        description: If applicable, [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the issuing country of the sender's identification
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/03_CardPayoutRequest_SenderAccountHolderGovernment.yaml:115]:
```suggestion
        description: If applicable, the [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the issuing country of the sender's identification
```

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/03_CardPayoutRequest_SenderAccountHolderGovernment.yaml [hunk 15]
```diff
@@ -85,14 +109,14 @@ properties:
...
         minLength: 2
         maxLength: 2
       date_of_expiry:
         type: string
-        description: The sender's ID expiry date. Format 'yyyy-MM-dd'
+        description: If applicable, the expiration date of the sender's identification. Format `yyyy-MM-dd`
```
Comment (eoin-shaughnessy-cko) [inline nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/03_CardPayoutRequest_SenderAccountHolderGovernment.yaml:120]: To be consistent, should it be either `YYYY-MM-DD` or `yyyy-mm-dd`?

**Review by eoin-shaughnessy-cko — COMMENTED** [review]
Hi @philip-wood-cko, thanks for sharing this! Great updates overall, just left some small comments. Give me a shout if you have any questions.

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/01_CardPayoutRequest_DestinationTypeCardAccountHolderIndividual.yaml (hunk 5 above)
Comment (philip-wood-cko) [reply nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/01_CardPayoutRequest_DestinationTypeCardAccountHolderIndividual.yaml]: updated.

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/03_CardPayoutRequest_SenderAccountHolderGovernment.yaml (hunk 15 above)
Comment (philip-wood-cko) [reply nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/03_CardPayoutRequest_SenderAccountHolderGovernment.yaml]: updated.

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/02_CardPayoutRequest_SenderAccountHolderCorporate.yaml (hunk 12 above)
Comment (philip-wood-cko) [reply nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/02_CardPayoutRequest_SenderAccountHolderCorporate.yaml]: updated.

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/03_CardPayoutRequest_DestinationTypeCardAccountHolderGovernment.yaml [hunk 16]
```diff
@@ -69,16 +98,16 @@ properties:
...
       issuing_country:
         type: string
-        description: The [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the payout destination owner's ID country of issue
+        description: If applicable, [two-letter ISO country code](https://checkout.com/docs/four/resources/codes/country-codes) of the issuing country of the payout destination owner's identification
         maxLength: 2
         minLength: 2
       date_of_expiry:
         type: string
-        description: The payout destination owner's ID expiry date. Format 'yyyy-MM-dd'
+        description: If applicable, the expiration date of the payout destination owner's identification. Format `yyyy-MM-dd`
```
[the remark this answers is not in the document]
Comment (philip-wood-cko) [reply nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/03_CardPayoutRequest_DestinationTypeCardAccountHolderGovernment.yaml]: updated.

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/01_CardPayoutRequest_SenderAccountHolderIndividual.yaml [hunk 17]
```diff
@@ -107,17 +121,30 @@ properties:
...
-        description: The sender's ID expiry date. Format 'yyyy-MM-dd'
+        description: If applicable, the expiration date of the sender's identification. Format `yyyy-MM-dd`
         maxLength: 10
         minLength: 10
+  date_of_birth:
+    type: string
+    description: |
+      The sender's date of birth
+      
+      Format `yyyy-MM-dd`
```
[the remark this answers is not in the document]
Comment (philip-wood-cko) [reply nas_spec/components/schemas/CardPayouts/CardPayoutRequest_SenderAccountHolderTypes/01_CardPayoutRequest_SenderAccountHolderIndividual.yaml]: updated.

File: nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/02_CardPayoutRequest_DestinationTypeCardAccountHolderCorporate.yaml (hunk 6 above)
Comment (philip-wood-cko) [reply nas_spec/components/schemas/CardPayouts/CardPayoutRequest_DestinationAccountHolderTypes/02_CardPayoutRequest_DestinationTypeCardAccountHolderCorporate.yaml]: updated.

**Comment by philip-wood-cko** [comment]
> Hi @philip-wood-cko, thanks for sharing this! Great updates overall, just left some small comments. Give me a shout if you have any questions.

Hey @eoin-shaughnessy-cko I've committed all your suggestions.

**Review by eoin-shaughnessy-cko — APPROVED** [review]
Thanks, @philip-wood-cko! Feel free to merge.

**Comment by philip-wood-cko** [comment]
> Thanks, @philip-wood-cko! Feel free to merge.

I don't have write access, so you'll need to merge it for me - happy for you to do it any time.

**Pull request merged**